delivery_carrier_label_paazl/models/stock_picking.py

```diff
@@ -214,7 +214,6 @@ def _paazl_commit_order_data(self):
                     product["quantity"] * product["weight"]
                     for product in self._paazl_order_data()["products"]["product"]
                 ),
-                description=self.note or "/",
```

Comment:
Can you motivate this change for the people who already use this module and will need to upgrade it?

Reply:
What do you mean?

Reply:
I mean that the people who will watch this PR will not understand why this change was needed.

Reply:
sure